arrange_schedule.py

```diff
@@ -1315,73 +1350,27 @@ def main():
         
         #crawl cwb radar image
         if raw_time >= alarm_crawler_cwb_img:
-            print("#6 "+ time.strftime('%Y-%m-%dT%H:%M:%SZ',now_time))
-            try:
-                newpid = os.fork()
-                if newpid == 0: #child
-                    shutdown = 1
-                    send_obj["server_dir"] = board_py_dir
-                    send_obj["user_id"] = 1
-                    receive_obj = crawler_cwb_img(send_obj)
-                    if receive_obj["result"] == "success":
-                        "DO NOTHING"
-                    else :
-                        receive_obj["error"] = "crawler_cwb_img : " + receive_obj["error"]
-                        set_system_log(receive_obj)
-                    os._exit(0)
-                else: #Parent
-                    alarm_crawler_cwb_img = raw_time + 3600.0
-            except:
-                receive_obj["result"] = "fail"
-                receive_obj["error"] = "fork4 error"
-                set_system_log(receive_obj)
-                alarm_crawler_cwb_img = raw_time + 600.0
+            fork_failed = cwb_crawler_worker.do(time.strftime('%Y-%m-%dT%H:%M:%SZ',now_time))
+            if fork_failed:
+                alarm_crawler_google_drive_img += 600.0
+            else:
+                alarm_crawler_google_drive_img += 3600.0
 
         #google calendar add to text data
         if raw_time >= alarm_google_calendar_text:
-            print("#7 "+ time.strftime('%Y-%m-%dT%H:%M:%SZ',now_time))
-            try:
-                newpid = os.fork()
-                if newpid == 0: #child
-                    shutdown = 1
-                    receive_obj = google_calendar_text()
-                    if receive_obj["result"] == "success":
-                        "DO NOTHING"
-                    else:
-                        receive_obj["error"] = "google_calendar_text" + receive_obj["error"]
-                        set_system_log(receive_obj)
-                    os._exit(0)
-                else: #Parent
-                    alarm_google_calendar_text = raw_time + 43200.0
-            except:
-                receive_obj["result"] = "fail"
-                receive_obj["error"] = "fork5 error"
-                set_system_log(receive_obj)
-                alarm_google_calendar_text = raw_time + 10.0
+            fork_failed = google_calendar_worker.do(time.strftime('%Y-%m-%dT%H:%M:%SZ',now_time))
+            if fork_failed:
+                alarm_google_calendar_text += 15.0
```

Comment:
why does this place add 15 rather than 10?

Reply:
Sorry, I mistakenly change it to 15 seconds.